src/plugins/intel_cpu/src/emitters/jit_snippets_emitters.cpp

```diff
@@ -554,7 +545,24 @@ template <dnnl::impl::cpu::x64::cpu_isa_t isa>
 void StoreEmitter::emit_isa(const std::vector<size_t> &in, const std::vector<size_t> &out) const {
     if (!store_emitter)
         IE_THROW() << "Store CPU emitter isn't initialized for StoreEmitter!";
+    using Vmm = typename dnnl::impl::utils::conditional3<isa == dnnl::impl::cpu::x64::sse41,
+            Xmm, isa == dnnl::impl::cpu::x64::avx2, Ymm, Zmm>::type;
+    /* When store_size > 16, the input Ymm register will not be
+    * preserved due to the usage of vextracti128 instruction.
+    */
+    // todo: is it better/faster to save it to a spare reg?
+    const bool input_not_preserved = !mayiuse(avx512_core) && count * dst_prc.size() > 16;
```

Comment:
[PR](https://github.com/openvinotoolkit/openvino/pull/16313#pullrequestreview-1341713049)

Reply:
The fix has been merged and the work around has been removed